libs/openchallenges/ui/src/lib/search-dropdown-filter/search-dropdown-filter.component.html

```diff
@@ -8,6 +8,11 @@
   [overlayOptions]="overlayOptions"
   [filter]="filter"
   (onFilter)="onSearch($event)"
+  [virtualScroll]="true"
+  [virtualScrollItemSize]="50"
```

Comment:
Is this the number of new item fetch during scrolling? The PR changelogs mentions that the initial number of items loaded is 25. Where is this value defined?

Reply:
This is the number to define the item height in px in the virtual viewpoint, which used to calculate how many items to render based on the scroll position and the viewport size.